source/mainnet/net/resources/release-notes.rst

```diff
@@ -11,6 +11,12 @@ Release Notes - Testnet
 
    To subscribe to updates on the Testnet status page click **Subscribe** to get all updates or click **Get updates** to choose to get all updates or only updates for specific products.
 
+January 19, 2023
+
+Cargo concordium 2.6.0
+---------------------------
+
+
```

Comment:
This should be removed for now, right?

Reply:
I have not checked it out locally. I just see it in the diff here (on github) that's why I commented.

But if it is gone then all is good. :)

Reply:
I forgot it was in the testnet release notes too. Now it is gone.